src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -59,6 +59,13 @@ module sonic-vxlan {
                 leaf dst_ip {
                     type inet:ip-address;
                 }
+
+                leaf ttl_mode {
```

Comment:
A minor enhancement here is define a custom type in sonic-types.yang for ttl_mode and use it in this yang as well as sonic-tunnel.yang.
You can do it in this PR or in a follow up PR